source/time_tables/thermometer.cpp

```diff
@@ -140,7 +140,7 @@ struct VertexProperties
     VertexProperties() : idx(0) {}
     VertexProperties(type::idx_t const& i): idx(i) {}
 };
-using Graph = boost::adjacency_list<boost::vecS, boost::vecS, boost::directedS, boost::property<boost::vertex_bundle_t, VertexProperties> >;
+using Graph = boost::adjacency_list<boost::vecS, boost::vecS, boost::directedS, VertexProperties>;
```

Comment:
OK for me, if that don't break tests.

Sample code from stack_overflow probably ^^

SouthWest french devs are so lazy you know ...